db/db.go

```diff
@@ -2354,17 +2367,18 @@ func GetWithdrawals(query string, length, start uint64, orderBy, orderDir string
 		if uiQuery, parseErr := strconv.ParseUint(query, 10, 64); parseErr == nil {
 			searchQuery := `
 				WHERE w.validatorindex = $3
-					OR block_slot = $3
-					OR (block_slot / $5) = $3
-					OR address_text LIKE ($4 || '%')`
```

Comment:
**question:** Why was the address search removed here? Are there not valid addresses that can be parsed as **uint64**?

Reply:
has not been removed, but we only search for exact matches